### PR TITLE
Global variables checker

### PR DESCRIPTION
**Pull request by shweta97**
Created and tested a custom checker that forbids the use of the global and nonlocal statements

### CODE REVIEW

File: examples/global_variables_test.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""Test file for global variables checker
+"""
+
+
+def method():
```
Comment (david-yz-liu) [inline examples/global_variables_test.py]: Use the term "function", not "method". Methods are functions which are defined in a class, bound to an object.

**Comment by david-yz-liu** [comment]
@shweta97 this is an excellent start, and I will merge it in. However, we also want to check for all top-level variables which are not constants. This is harder to do, and is what I want you to work on next.

**Comment by shweta97** [comment]
I've been trying to do a check for top-level variables with no luck. I will continue to work on it.

File: pyta.py [hunk 2]
```diff
@@ -30,6 +30,9 @@ def check(module_name):
     linter = lint.PyLinter(reporter=reporter)
     linter.load_default_plugins()
     linter.load_plugin_modules(['checkers'])
+    linter.load_plugin_modules(['global_variables_checker'])
```
Comment (david-yz-liu) [inline pyta.py]: I think you can add this module to the list in the previous line

File: pyta.py [hunk 3]
```diff
@@ -30,6 +30,9 @@ def check(module_name):
...
+    linter.read_config_file()
```
Comment (david-yz-liu) [inline pyta.py]: Make sure to do a pull from master.

Comment (shweta97) [inline pyta.py]: I've tried that but there are some conflicts arising and I'm not sure where I can see what the conflicts are.

**Comment by shweta97** [comment]
I'm still trying to differentiate the top-level constants from the variables using a bit of regex. But i've committed the rest of it what I have done this week.

**Comment by david-yz-liu** [comment]
@shweta97 You have made the same mistake as @Jasminewwwu: adding too many changes to the same pull request.

You should now create two new branches, one for each new checker you added, and make two different pull requests.

**Pull request closed without merging**